src/EditorFeatures/Core/Classification/Syntactic/SyntacticClassificationTaggerProvider.TagComputer.cs

```diff
@@ -365,33 +363,35 @@ async ValueTask<SnapshotSpan> ComputeChangedSpanAsync()
                 return _lastProcessedData;
         }
 
-        public SegmentedList<ITagSpan<IClassificationTag>> GetTags(NormalizedSnapshotSpanCollection spans)
+        public void AddTags(NormalizedSnapshotSpanCollection spans, SegmentedList<ITagSpan<IClassificationTag>> tags)
         {
             _taggerProvider._threadingContext.ThrowIfNotOnUIThread();
 
             using (Logger.LogBlock(FunctionId.Tagger_SyntacticClassification_TagComputer_GetTags, CancellationToken.None))
-            {
-                return GetTagsWorker(spans) ?? new();
-            }
+                AddTagsWorker(spans, tags);
         }
 
-        private SegmentedList<ITagSpan<IClassificationTag>>? GetTagsWorker(NormalizedSnapshotSpanCollection spans)
+        private void AddTagsWorker(NormalizedSnapshotSpanCollection spans, SegmentedList<ITagSpan<IClassificationTag>> tags)
         {
             _taggerProvider._threadingContext.ThrowIfNotOnUIThread();
             if (spans.Count == 0 || _workspace == null)
-                return null;
+                return;
 
             var snapshot = spans[0].Snapshot;
 
             if (TryGetClassificationService(snapshot) is not (var solutionServices, var classificationService))
-                return null;
+                return;
 
             using var _ = Classifier.GetPooledList(out var classifiedSpans);
 
             foreach (var span in spans)
                 AddClassifications(span);
 
-            return ClassificationUtilities.Convert(_taggerProvider._typeMap, snapshot, classifiedSpans);
+            var typeMap = _taggerProvider._typeMap;
+            foreach (var classifiedSpan in classifiedSpans)
+                tags.Add(ClassificationUtilities.Convert(typeMap, snapshot, classifiedSpan));
```

Comment:
>Convert [](http://example.com/codeflow?start=49&length=7)

Is the SegmentedList version of this method used anywhere now?

Reply:
Will check

Reply:
nope.  removed.